src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PeopleTable } from '../../components/PeopleTable';
+import { PeopleFilters } from '../../components/PeopleFilters';
+import { useSearchParams } from 'react-router-dom';
+import { UrlSearchParams } from '../../types/UrlSearchParams';
+import { SortTypes } from '../../types/SortTypes';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [searchParams] = useSearchParams();
+  const query = searchParams.get(UrlSearchParams.Query) || '';
+  const normalizedQuery = query.trim().toLowerCase();
+  const sexFilter = searchParams.get(UrlSearchParams.Sex) || '';
+  const centuries = useMemo(
+    () => searchParams.getAll(UrlSearchParams.Centuries),
+    [searchParams],
+  );
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      try {
+        const fetchedPeople = await getPeople();
+
+        setPeople(fetchedPeople);
+      } catch (e) {
+        setIsError(true);
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    fetchData();
+  }, []);
+
+  const sortByAlphabet = (
+    arr: Person[],
+    key: SortTypes,
+    direction: string = 'asc',
+  ) =>
+    arr.sort((a, b) =>
+      direction === 'desc'
+        ? String(b[key]).localeCompare(String(a[key]))
+        : String(a[key]).localeCompare(String(b[key])),
+    );
+
+  const sortByNumber = (
+    arr: Person[],
+    key: SortTypes,
+    direction: string = 'asc',
+  ) =>
+    arr.sort((a, b) =>
+      direction === 'desc' ? +b[key] - +a[key] : +a[key] - +b[key],
+    );
```

Comment:
The sortByNumber function assumes that the keys are always numbers. Ensure that the data being sorted is consistently of type number to avoid unexpected behavior.